app/_src/kubernetes-ingress-controller/guides/using-external-service.md

```diff
@@ -2,45 +2,43 @@
 title: Expose an external application
 ---
 
-This example shows how we can expose a service located outside the Kubernetes cluster using an Ingress.
+Learn to expose a service located outside the Kubernetes cluster using an Ingress.
 
-{% include_cached /md/kic/installation.md kong_version=page.kong_version %}
-
-{% include_cached /md/kic/class.md kong_version=page.kong_version %}
+{% include_cached /md/kic/prerequisites.md kong_version=page.kong_version disable_gateway_api=false %}
 
 ## Create a Kubernetes Service
 
-First we need to create a Kubernetes Service [type=ExternalName][0] using the hostname of the application we want to expose:
+1. Deploy a Kubernetes Service [type=ExternalName][0] using the hostname of the application you want to expose.
 
-```bash
-echo "
-kind: Service
-apiVersion: v1
-metadata:
-  name: proxy-to-httpbin
-spec:
-  ports:
-  - protocol: TCP
-    port: 80
-  type: ExternalName
-  externalName: httpbin.org
-" | kubectl apply -f -
-```
-Response:
-```
-service/echo created
-```
+    ```bash
+    echo "
+    kind: Service
+    apiVersion: v1
+    metadata:
+      name: proxy-to-httpbin
+    spec:
+      ports:
+      - protocol: TCP
+        port: 80
+      type: ExternalName
+      externalName: httpbin.org
+    " | kubectl apply -f -
+    ```
+    The results should look like this:
+    ```
+    service/proxy-to-httpbin created
+    ```
 
-## Create an Ingress to expose the service at the path `/httpbin`
+1. Create an Ingress to expose the service at the path `/httpbin`
 
-{% include_cached /md/kic/http-test-routing-resource.md kong_version=page.kong_version path='/httpbin' name='proxy-from-k8s-to-httpbin' service='proxy-to-httpbin' port='80' %}
+    {% include_cached /md/kic/http-test-routing-resource.md kong_version=page.kong_version path='/httpbin' name='proxy-from-k8s-to-httpbin' service='proxy-to-httpbin' port='80' %}
 
 ## Test the Service
 
 ```bash
 curl -si http://kong.example/httpbin/anything --resolve kong.example:80:$PROXY_IP
```

Comment:
This should use the `Host` header instead

Reply:
Thank you!!! I missed that. now comitted the changes.